portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,22 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <div id ="header">
```

Comment:
Did you want this to go under < body >?

Reply:
No this was correct

Reply:
Also I did test it. It wirked

Reply:
Hmm, I'm very surprised by this.

< head > is usually reserved for metadata elements.

Some context:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head
https://stackoverflow.com/questions/26088555/placing-a-div-in-head

It could be browsers doing weird things and fixing up the html markup, even though it's standards compliant... though looking at this again, it looks like you have the same text on line 14. Are you sure it's not showing that instead?

Reply:
Hmm! This is surprising to me, because < head> is reserved for metadata elements. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head  and https://stackoverflow.com/questions/26088555/placing-a-div-in-head for some context.

It could be browsers trying to proactively fix the markup, but looking at this...you have a similar line on line 14.

Are you seeing two "Sean Tisdale Portfolio"? Or is it possible that only the one on line 14 is showing up?